benchmarking/rotational_diff_visualizer.py

```diff
@@ -0,0 +1,119 @@
+"""
+Analyzing the error distribution of oblique tag observations v.s. straight-on tag observations.
```

Comment:
Following up on the previous comment, this is one of those scripts that seems like it has a lot of overlap with other scripts.